src/components/manage/Widgets/DatetimeWidget.jsx

```diff
@@ -73,22 +86,26 @@ class DatetimeWidget extends Component {
   constructor(props) {
     super(props);
 
-    let datetime = moment().utc();
+    let datetime = null;
 
     if (this.props.value) {
       // check if datetime has timezone, otherwise assumes it's UTC
       datetime = this.props.value.match(/T(.)*(-|\+|Z)/g)
-        ? moment(this.props.value).utc()
-        : moment(`${this.props.value}Z`).utc();
+        ? // Since we assume UTC everywhere, then transform to local (momentjs default)
+          moment(this.props.value)
+        : // This might happen in old Plone versions dates
+          moment(`${this.props.value}Z`);
     }
 
-    if (!this.props.value && this.props.dateOnly) {
-      datetime.set(defaultTimeDateOnly);
-    }
+    // @nzambello do we need this if using null by default?
```

Comment:
@nzambello why was this here? Any use case?

Reply:
If we are using null as default it's not necessary